src/main/java/org/web3j/console/project/ProjectImporterCLIRunner.java

```diff
@@ -31,15 +31,24 @@
 
     @Override
     public void run() {
-        if (InputVerifier.requiredArgsAreNotEmpty(projectName, packageName, solidityImportPath)
-                && InputVerifier.classNameIsValid(projectName)
-                && InputVerifier.packageNameIsValid(packageName)) {
-            try {
-                new ProjectImporter(outputDir, packageName, projectName, solidityImportPath)
-                        .generate();
-            } catch (final Exception e) {
-                exitError(e);
+        if (InputVerifier.requiredArgsAreNotEmpty(projectName, packageName, solidityImportPath)) {
+            if (InputVerifier.classNameIsValid(projectName)) {
+                if (InputVerifier.packageNameIsValid(packageName)) {
```

Comment:
This nesting is a bit deep.. Think it would be cleaner to check them individually and do an exitError if they don't pass, before continuing, if that makes sense?